MarkBind/vue-strap
# Remove unused components

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Other, please explain: Migration



Part of MarkBind/markbind#266



**What is the rationale for this request?**
Components that are not used by us should be removed rather than migrated.

**What changes did you make? (Give an overview)**
Each commit removes a component mentioned in the commit title. Documentation content for other components may be modified if it uses the removed component.

**Provide some example code that this change will affect:**
NIL

**Is there anything you'd like reviewers to focus on?**
NIL

**Testing instructions:**
Ensure that it still compiles and runs without error (`npm run build` and `npm run docs`).
